asciidoc-confluence-publisher-maven-plugin/src/main/java/org/sahli/asciidoc/confluence/publisher/maven/plugin/AsciidocConfluencePublisherMojo.java

```diff
@@ -82,6 +87,16 @@
     @Parameter(property = PREFIX + "password")
     private String password;
 
+    // support settings.xml
+    @Parameter(readonly = true, property = "settings")
+    protected org.apache.maven.settings.Settings mavenSettings;
+
+    @Parameter(property = PREFIX + "serverId")
+    private String serverId;
```

Comment:
Why is this property called `serverId`? Would you see another name with more semantics, e.g. something related to `credentialSettings` or `externalCredentials`, or the like?

Reply:
Inside settings.xml, we are referencing the following entry
```
    <server>
      <id>myServerId</id>
      <username>myusername</username>
      <password>myEncryptedPassword</password>
    </server>
```
since it is server/id - I named the property serverId. Suggest we keep it as is.